data-access-layer/db/DBManager/DBManager.js

```diff
@@ -69,8 +69,7 @@ class DBManager {
         logger.info(`Connecting to DB with the provided config URL : ${config.mongodb.url}`);
         return this.initDb(config.mongodb);
       }
-    }) // On deterministic error, just log message and stop
-      .catch(ServiceBindingNotFound, err => logger.warn('MongoDB binding to ServiceFabrik not found. This generally should not occur. More Info:', err))
```

Comment:
Is this related to this change?

Reply:
Earlier `ServiceBindingNotFound` was thrown in `getBindingProperty` at [LN 127](https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/665/files#diff-637f9dd65cf53fe8e7cbf23bd6469127L127) below. Now we use `NotFound` from ApiServer instead. This scenario will mainly come in picture at the time of landscape recreation/scratch setup.